src/compiler/checker.ts

```diff
@@ -10486,8 +10484,6 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         switch (propertyName) {
             case TypeSystemPropertyName.Type:
                 return !!getSymbolLinks(target as Symbol).type;
-            case TypeSystemPropertyName.EnumTagType:

```

Comment:
Is this name also obsolete?

Reply:
Yes, the name has also been removed in the enum declaration.

Reply:
If it has, you may not have pushed it, as I don't see that in the diff.

Reply:
Oh oops, sorry, I though this was an import 
![image](https://github.com/microsoft/TypeScript/assets/5341706/3d2e8e07-9b04-418e-8dc8-fb1ba3e7d2ba)

Reply:
I had that _exact_ problem when going over the diff myself :)